test/optim/test_optimize.py

```diff
@@ -930,25 +950,25 @@ def nlc4(x):
                     acq_function=mock_acq_function,
                     bounds=bounds,
                     q=1,
-                    nonlinear_inequality_constraints=[nlc1],
+                    nonlinear_inequality_constraints=[(nlc1, True)],
                     num_restarts=num_restarts,
                     batch_initial_conditions=4 * torch.ones(1, 1, 3, **tkwargs),
                 )
             # Explicitly setting batch_limit to be >1 should raise
-            with self.assertRaisesRegex(
-                ValueError,
-                "`batch_limit` must be 1 when non-linear inequality constraints "
-                "are given.",
-            ):
-                optimize_acqf(
-                    acq_function=mock_acq_function,
-                    bounds=bounds,
-                    q=1,
-                    nonlinear_inequality_constraints=[nlc1],
-                    batch_initial_conditions=torch.rand(5, 1, 3, **tkwargs),
-                    num_restarts=5,
-                    options={"batch_limit": 5},
-                )
+            # with self.assertRaisesRegex(
```

Comment:
lets add this and the restriction back in for now given my comment above

Reply:
Done.